Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/ExplicitInitConfiguration.swift

```diff
@@ -0,0 +1,23 @@
+struct ExplicitInitConfiguration: SeverityBasedRuleConfiguration, Equatable {
+    typealias Parent = ExplicitInitRule
+
+    @ConfigurationElement(key: "severity")
+    private(set) var severityConfiguration = SeverityConfiguration<Parent>(.warning)
+    @ConfigurationElement(key: "include_explicit_init")
+    private(set) var includeExplicitInit = true
+    @ConfigurationElement(key: "include_bare_init")
```

Comment:
Swift calls expression like `.init(...)` "implicit member expression". Not sure we should use the term in an **Explicit**InitRule, though. So "bare" might be okay. What do you think?

Reply:
So I'm not super-happy with "bare". "inferred" sounds a little bit better, and is at least in keeping with existing terms of art, but "inferred_init" doesn't feel quite right, as it's not the "init" that is inferred ... totally open to suggestions if anyone has something better.

Reply:
I'm struggling as well. `include_implict_init_members` maybe?

Reply:
Maybe `bare` is ok - "implicit member expression" feels a little obscure

Reply:
The name would not matter so much if there was a description added to each option. The infrastructure is there already. I may prepare a PR in the upcoming days to open it up and add a few examples. More can then be attached over time.

Long story short, let's keep "bare". 😉